collections/0xethereals/inscriptions.json

```diff
@@ -0,0 +1,210 @@
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "4"
+    },
+    {
+        "id": "0x9b068349dbb36f0cd58de5e9ab2f1077270181b705134182411a284b038652fa",
+        "meta": {
+            "name": "0xEthereal #5",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "5"
+    },
+    {
+        "id": "0x83d3ff85d942c2fcf7840fb5aed4928f1eb448f9d4fb1caeaab2ebc405315698",
+        "meta": {
+            "name": "0xEthereal #6",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "6"
+    },
+    {
+        "id": "0x8c2445c1e4a11f7b4f4f4598cdb863c19509bba854ab8fb0fb24b278886059b5",
+        "meta": {
+            "name": "0xEthereal #7",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "7"
+    },
+    {
+        "id": "0xbe765d180b01e2e5ab451e4b974b584ea96c1470c05a1ffa77d1aa856f45cd28",
+        "meta": {
+            "name": "0xEthereal #8",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "8"
+    },
+    {
+        "id": "0xf4e7ba81909f7f0cf47ee807c08b1e7ce440aa4cbc7d03c4a25341a8b1976d6a",
+        "meta": {
+            "name": "0xEthereal #9",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "9"
+    },
+    {
+        "id": "0x34450f6dbfaa8b4ac85e17510d556a5d865539fa0162687b63469c36aa20e029",
+        "meta": {
+            "name": "0xEthereal #A",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "10"
+    },
+    {
+        "id": "0x2737c8530b94aefa99bb5bc41ffc1fbdff6467328f6d952df29e38a8ea02db3e",
+        "meta": {
+            "name": "0xEthereal #B",
+            "attributes": [
+                {
+                    "trait_type": "collection",
+                    "value": "0xEthereals"
+                }
+            ]
+        },
+        "number": "11"
+    },
+    {
+        "id": "0xe54aeeadcf113a69bec99fc3da22c70bc67518ba9d217fad620659853d6ff2b9",
+        "meta": {
+            "name": "0xEthereal #C",
+            "attributes": [
```

Comment:
@0xethereals  As a native marketplace, we don't need such collection traits, unlike Emblem Vault.  Add unique traits per item if you want us automatically generate rarity rank for your collection. if you don't, make this "attributes" field an empty array [].

Reply:
Awesome, we saw EthPixels with this "collection" trait in your repo and decided to add it. Will add different trait, thank you.